dz2.2/dz2.2.ts

```diff
@@ -0,0 +1,389 @@
+class School {
+    // implement 'add area', 'remove area', 'add lecturer', and 'remove lecturer' methods
```

Comment:
I can't find the realization of 'add lecturer' and 'remove lecturer' methods.

Reply:
Added.